### PR TITLE
Chore: formatted the date and time and addedd icons to it

### PR DESCRIPTION
**Pull request by Yonela-Johannes**
... and then formatted the code

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def new
   def create
     @event = Event.new(event_params)
     if @event.save
-      redirect_to root_path, notice: "Event was successfully created." # please could we redicrect to @event page if an event is successfully created?
+      redirect_to event_path @event, notice: "Event was successfully created." # please could we redicrect to @event page if an event is successfully created?
```
Comment (agoschen) [inline app/controllers/events_controller.rb:13]: Does it work like this? Dote it not need to have @event in brackets? redirect_to event_path(@event)

**Review by agoschen — CHANGES_REQUESTED** [review]
@mojafa @Yonela-Johannes have added comments :)

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (Yonela-Johannes) [reply app/controllers/events_controller.rb:13]: Yes, it does with ruby you don't need the parenthesis there if it is one param.

File: app/views/events/show.html.erb [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
-  <p>Total Budget: R<%= @event.total_budget %></p>
-
-
-<%= link_to "View tasks & create a new task", event_tasks_path(@event)%>
-<%= link_to "Add expenses", new_event_expense_path(@event) %>
-<%= link_to "Guests", event_guests_path(@event)%>
-
+  <p>Total Budget: R<%= @event.total_budget.round(2) %></p>
+  <%= link_to "View tasks & create a new task", event_tasks_path(@event)%>
+  <%= link_to "Add expenses", new_event_expense_path(@event) %>
```
[the remark this answers is not in the document]
Comment (mojafa) [reply app/views/events/show.html.erb:21]: yes. Checking this branch out locally before i approve the merge.

**Pull request merged**